modules/nf-core/scanpy/filter/tests/main.nf.test

```diff
@@ -0,0 +1,70 @@
+nextflow_process {
+
+    name "Test Process SCANPY_FILTER"
+    script "../main.nf"
+    process "SCANPY_FILTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "scanpy"
+    tag "scanpy/filter"
+
+    test("Should run without failures") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                        [ id: 'test' ],
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/scrnaseq/h5ad/SRR28679756_filtered_matrix.h5ad', checkIfExists: true)
+                    ]
+                )
+                input[1] = 20
+                input[2] = 20
+                input[3] = 50
+                input[4] = 50
+                input[5] = 50
+                """
+            }
+        }
+
+        then {
+            assertAll(
+            { assert process.success },
+            { assert snapshot(process.out).match() }
```

Comment:
Could you fix the alignment and add, `path(process.out.versions[0]).yaml` to the snapshot ?